src/main/java/jenkinsci/plugins/influxdb/InfluxDbPublicationService.java

```diff
@@ -294,33 +320,7 @@ private void addPointsFromPlugin(List<Point> pointsToWrite, PointGenerator gener
         }
     }
 
-    private InfluxDBClient getInfluxDBClient(Run<?, ?> build, Target target, URL url) {
-        StandardUsernamePasswordCredentials credentials = CredentialsProvider.findCredentialById(target.getCredentialsId(), StandardUsernamePasswordCredentials.class, build);
-        InfluxDBClient influxDB;
-
-        if (target.getOrganization() != null && !target.getOrganization().trim().isEmpty()) {
-            InfluxDBClientOptions.Builder options = InfluxDBClientOptions.builder()
-                    .url(target.getUrl())
-                    .org(target.getOrganization())
-                    .bucket(target.getDatabase())
-                    .okHttpClient(createHttpClient(url, target.isUsingJenkinsProxy()));
```

Comment:
I reviewed my code again and I think I might broke the proxy feature. But this is the only section in the code that used the proxy, anyways. And in the `Target` class, no proxy was used. And in my understanding, this check was for the Jenkins settings (connection check).

Reply:
I just did a search in the repo about this. Looks like you're right. To my understanding, the v2 client uses proxy, but v1 client doesn't. It would be pretty simple to add the proxy support to the new client wrapper you introduce here at least for the v2 client.

This is one of those features that I haven't actually needed myself, it was implemented in a PR. There hasn't been any bug reports about this _not_ working, so it has gone under the radar until now. And a proxy setting is pretty difficult to test with unit tests as all endpoints are mocked anyway. Would you be able to fix the proxy config with this PR as well?